widget/select.go

```diff
@@ -333,17 +336,20 @@ func (s *selectRenderer) MinSize() fyne.Size {
 	s.combo.propertyLock.RLock()
 	defer s.combo.propertyLock.RUnlock()
 
-	min := fyne.MeasureText(s.combo.PlaceHolder, theme.TextSize(), s.combo.textStyle())
-
-	min = min.Add(fyne.NewSize(theme.Padding()*6, theme.Padding()*4))
-	return min.Add(fyne.NewSize(theme.IconInlineSize()+theme.Padding(), 0))
+	// x1			      x4 (reserved)          x1    x1     x2
+	// pad  -     label (placeholder)  -  pad - icon - pad*2
```

Comment:
I’m not sure what these comments mean - should they be removed?

Reply:
I can removed, that comment is just to show how the MinSize is build, but it does not have any importance